lib/customer.rb

```diff
@@ -0,0 +1,56 @@
+require_relative 'twilio'
+
+class Customer
+
+  attr_reader :order, :menu, :send_text
+
+  def initialize(menu = Menu.new, send_text = TwilioSend.new)
+    @order = {}
+    @menu = menu.dishes
+    @send_text = send_text
+  end
+
+  def make_order(dish, quantity)
+    fail "Sorry! That dish is not on the menu" unless dish_available?(dish)
+    if order.include?(dish)
+      order[dish] += quantity
+    else
+      order.store(dish, quantity)
+    end
+  end
+
+  def total_dishes
+    order.values.reduce(:+)
+  end
+
+  def order_summary
+    "You have ordered:"
+    order.map { |dish, quantity| "#{quantity} #{dish}" }.join(", ")
+  end
+
+  def charge
+    "You've ordered #{total_dishes} dishes and cost is £#{total_cost}0."
+  end
+
+  def confirm_order(customer_response)
+    "Would you like confirm your order? y/n"
+    customer_response = customer_response.downcase
+    if customer_response == "y"
+      send_text.sms(+447514209436)
```

Comment:
Style/NumericLiterals: Separate every 3 digits in the integer portion of a number with underscores(_).